README.md

```diff
@@ -48,6 +49,7 @@ bin/behat -f cucumber_json
 - `fileName` _(optional)_: Filename of generated report - current feature name will be used by default.
 Only applicable when `resultFilePerSuite` is not enabled.
 - `resultFilePerSuite` _(optional)_: The default behaviour is to generate a single report named `all.json`.
+- `screenshotExtension` _(optional)_: The name of the extension to be used to take screenshots.
```

Comment:
This line should be a line lower. The next line ("If this option is set to `true`..." belongs to the `resultFilePerSuite` option, not to the `screenshotExtension`.

Reply:
Fixed